healthcheck/sqlhealthcheck.go

```diff
@@ -155,3 +155,35 @@ func EvaluateHealthCheck(healthCheck SQLHealthCheck) (err HCError) {
 	return err
 
 }
+
+//Implementation of Reportable Element
+
+//HealthCheckReportHeaders headers used for GetHeaders
+var HealthCheckReportHeaders = []string{"Title", "Query", "Passed", "Expected", "Actual"}
```

Comment:
A small excerpt from effective go:
"Constants in Go are just that—constant. They are created at compile time, even when defined as locals in functions, and can only be numbers, strings or booleans. Because of the compile-time restriction, the expressions that define them must be constant expressions, evaluatable by the compiler."

I attempted to make this array constant at first, but arrays are not made until runtime, and therefore can't be constant. The alternative would be to make each string in that array it's own constant